bot/exts/evergreen/pythonfacts.py

```diff
@@ -0,0 +1,23 @@
+import random
+
+import discord
+from discord.ext import commands
+from discord.ext.commands.bot import Bot
+
+
+class PythonFacts(commands.Cog):
+    """Gives a random fun fact about Python."""
+
+    def __init__(self, bot: Bot) -> None:
```

Comment:
```suggestion
    def __init__(self, bot: commands.Bot) -> None:
```
To keep consistency with the setup function in this file, and other parts of the bot, we should change this.
Once this has been changed, we can also remove the `from discord.ext.commands.bot import Bot` line at the top of the file.

Reply:
OK